test/testQMLFile.js

```diff
@@ -21,7 +21,7 @@ if (!QMLFile) {
     var QMLFile = require("../QMLFile.js");
     var QMLFileType = require("../QMLFileType.js");
     var CustomProject =  require("loctool/lib/CustomProject.js");
-    var ResourceString =  require("loctool/lib/ResourceString.js");
+    var ContextResourceString =  require("loctool/lib/ContextResourceString.js");
```

Comment:
The tests can depend on the loctool because the loctool does not depend on the tests so there is no circular dependency.